scripts/config.toml

```diff
@@ -59,10 +57,9 @@ interval = "epoch"
 loss_property = ['per_molecule_energy', 'per_atom_force'] # use
 
 [training.loss_parameter.weight]
-per_molecule_energy = 0.999 #NOTE: reciprocal units
+per_molecule_energy = 0.009 #NOTE: reciprocal units
```

Comment:
This has different weights than the training toml.  Is that intentional or was this just from experimentation?  I think it would probably be good to have the defaults be the same in the two files (even if we ultimately decide to change the defaults).

Reply:
good catch. Will fix this!